00-RELEASENOTES

```diff
@@ -1,16 +1,165 @@
+SECURITY: There are security fixes in the release.
+--------------------------------------------------------------------------------
 
-There is no release notes for this branch, it gets forked into another branch
-every time there is a partial feature freeze in order to eventually create
-a new stable release.
 
-Usually "unstable" is stable enough for you to use it in development environments
-however you should never use it in production environments. It is possible
-to download the latest stable release here:
+================================================================================
+Valkey 8.1.0 RC1  -  Released Thu 11 Feb 2025
+================================================================================
+Upgrade urgency LOW: This is the first release candidate of Valkey 8.1, with
+performance improvements, extended observability and cluster improvements and different bug fixes. 
+It includes a new implementation of the Valkey dictionary which is more memory and cache efficient, 
+better performance for encryption in transit, reduced replication overhead by offloading work to I/O threads,
+faster failover support in cluster mode, major improvements to the active defrag process to reduce the impact on command processing
+and different API changes for improved usability. 
+Valkey now supports new new check-and-set feature for native STRINGs.
+In this release Valkey also supports CMake build system for enhanced building options and better IDE integration. 
 
-    https://valkey.io/download/
+API and Interface changes
+=========================
+* Introduce bgsave cancel (#757)
+* Add paused_reason to INFO CLIENTS (#1564)
+* show capa in CLIENT LIST / CLIENT INFO (#1698)
+* Set Command with IFEQ Support (#1324)
+* Add new flag in `CLIENT LIST` for import-source client (#1398)
+* Add more filters to CLIENT LIST (#1401)
+* Add `availability_zone` to the HELLO response (#1487)
 
-More information is available at https://valkey.io
+Observability and Monitoring changes
+====================================
+* Extend LATENCY LATEST to add sum / cnt stats (#1570)
+* Add paused_actions for INFO Clients (#1519)
+* Feature COMMANDLOG to record slow execution and large request/reply (#1294)
+* Fix cluster info sent stats for message with light header (#1563)
+* Add latency stats around cluster config file operations (#1534)
 
-Happy hacking!
+Performance/Efficiency Improvements - Core
+==========================================
+* New hash table (#1186)
+* Free strings during BGSAVE/BGAOFRW to reduce copy-on-write (#905)
+* Create an empty lua table with specified initial capacity as much as possible (#1092)
+* Move prepareClientToWrite out of loop for HGETALL command (#1119)
+* Improved hashing algorithm in luaS_newlstr (#1168)
+* Replace dict with new hashtable for sets datatype (#1176)
+* Do security attack check only when command not found to reduce the critical path. (#1212)
+* Accelerate hash table iterator with prefetching (#1501)
```

Comment:
Since these aren't ordered, maybe let's put all of the hash table ones together

Reply:
Regarding "Improved hashing algorithm in luaS_newlstr": this title is hard to get. It improves performance of Lua tables, optimizing Lua scripts in general.

Suggestion:

    * Improved hashing algorithm for Lua tables (#1168)